cmd/server/main.go

```diff
@@ -575,56 +573,26 @@ func initNamespace(config config.Configuration, namespaces ...namespace.Handler)
 	return namespaceManager, nil
 }
 
-func initPGClients(config config.PostgresConfig) (*struct {
-	driver                 *entDialectSQL.Driver
-	entitlementDBClient    *entitlementdb.Client
-	productcatalogDBClient *productcatalogdb.Client
-	creditDBClient         *creditdb.Client
-}, error,
+func initPGClients(config config.PostgresConfig) (
+	*entDialectSQL.Driver,
+	*db.Client,
+	error,
 ) {
```

Comment:
Can we please just return a named struct?

```suggestion

type pgClients struct {
    driver *entDialectSQL.Driver
	client *db.Client
}

func initPGClients(config config.PostgresConfig) (
	*pgClients,
	error,
) {
```

Reply:
IMO 3 return values is ok but sure